### ISSUE TITLE
No mouse support under vmware fusion

### ISSUE THREAD

**Issue by l0x2a** [REMnux/remnux-cli#3]
Hi all, please excuse me if this isn't the right place to ask. But I've just installed Remnux v7 as OVA on my Mac (Catalina 10.15.6) under VMWare Fusion 11.5.5 and I'm unabel to find a way to get the mouse working in the vm. I can see the mouse pointer and move it, but I cannot click on anything. I can type in the open terminal, but that's all. I've re-installed both open-vmware-tools packages and tried the original VMWare-Tools. Did remnux upgrade / update. No success! I can ssh in the vm as a workaround, but it would be nice to use the vm directly. If you have any ideas ... please let me know. Thanx in advance!

**Comment by lennyzeltser** [comment]
Sorry to hear about this issue! The setup you described should work fine, as confirmed through multiple tests.

A common reason for mouse not working with a VM in VMware Fusion is the macOS security mechanism that requires you to give Fusion the access it needs. Could you check the following:

`System Preferences` > `Security & Privacy` >  `Privacy` > `Accessibility`

Make sure that VMware Fusion has the enabled checkbox. If it's not enabled, enable it and restart Fusion.

**Comment by l0x2a** [comment]
Hi Lenny,
thanks for your reply. That was the first thing I've checked. I've got a whole lot of other VMs and their working fine. But ... now I've completly deleted Fusion.App from the Accessibility list and added it again and this seems to do the magic ;-) So, again, sorry for bothering you guys with such trivial problems AND let me tell you ... I love Remnux!!! It is such a great help in my daily work! Keep it up guys, you rock!
Cheers
